### DISCUSSION TITLE
Matplotlib Cheatsheet

### DISCUSSION THREAD

**Discussion by WEI-CHEN-CHUN** [1101-GS4719-C/Python-Community-GS4719-C#741] [General]
用Matplotlib畫圖，不用記很多語法，官方都整理好小抄了
包括各種style,color,colormap
畫出各種圖就靠這個了

https://matplotlib.org/cheatsheets/

**Comment by Chiway1229** [comment]
這個套件對我接下來的資訊科學導論與統計學感覺非常有用
好耶

**Comment by AnAnistired** [comment]
一年級微積分python進度真的超快，但完成作業後又很有成就感

**Reply by WEI-CHEN-CHUN** [reply]
用Matplotlib寫感覺比pylab好看，有較多可控的東西

**Comment by Yunhans** [comment]
統計學期中考如果能用這個畫圖就好了

**Comment by manipulateed** [comment]
這個可是資料視覺化常用的套件之一呢，用途也非常廣泛，值得當作儲備小技能。

**Comment by williamyen0623** [comment]
這個超讚 超好用 大家肯定要去試試看

**Comment by Pietra1226** [comment]
還沒有用過，但看了一下覺得很心動感覺很方便

**Comment by danwan1027** [comment]
這個整理太棒了！感謝分享

**Comment by evanchou1204** [comment]
感覺蠻讚的，而且他的圖應該會比ppt內建的好看很多！
